src/modules/init/index.ts

```diff
@@ -104,25 +104,29 @@ const promptTemplates = async (): Promise<string> => {
   return chosen
 }
 
-const promptContinue = async (repoName: string) => {
+const promptContinue = async (repoName: string, projectFolderName?: string) => {
   const proceed = await promptConfirm(
-    `You are about to create the new folder ${process.cwd()}/${repoName}. Do you want to continue?`
+    `You are about to create the new folder ${process.cwd()}/${projectFolderName ?? repoName}. Do you want to continue?`
   )
   if (!proceed) {
     log.info('Bye o/')
     process.exit()
   }
 }
 
-export default async () => {
+export default async (projectFolderName?: string) => {
   log.debug('Prompting for app info')
   log.info('Hello! I will help you generate basic files and folders for your app.')
   try {
     const { repository: repo, organization: org } = templates[await promptTemplates()]
-    await promptContinue(repo)
+    await promptContinue(repo, projectFolderName)
     log.info(`Cloning ${chalk.bold.cyan(repo)} from ${chalk.bold.cyan(org)}.`)
-    await git.clone(repo, org)
-    log.info(`Run ${chalk.bold.green(`cd ${repo}`)} and ${chalk.bold.green('vtex link')} to start developing!`)
+    await git.clone(repo, org, projectFolderName)
+    log.info(
```

Comment:
here

Reply:
Also instead of using `chalk.bold.green` how about using `ColorifyConstants` from `Colors.ts` ? 

Example: [here](https://github.com/vtex/toolbelt/blob/b4eba5ed9edbc6d67dcb3c55967905367e899787/src/lib/constants/Messages.ts#L9)